documentation/source/library_reference.rst

```diff
@@ -2,6 +2,21 @@
 Library Reference
 #################
 
+.. spelling::
+    BusDriver
+    bigEndian
+    BinaryRepresentation
+    binaryRepresentation
+    sim
+    SimHandle
+    Todo
+    un
+    arg
+    idx
+    Args
+    Enum
```

Comment:
Are all of these needed? Looks like some of them should be marked up instead. Or are they section titles?

Reply:
I now removed them and mysteriously spell checking still is fine, while it wasn't before... no idea why.